src/cosmic_ray/operators/boolean_replacer.py

```diff
@@ -15,7 +15,7 @@ class ReplaceTrueWithFalse(KeywordReplacementOperator):
 class ReplaceFalseWithTrue(KeywordReplacementOperator):
     """An that replaces False with True."""
     from_keyword = 'False'
-    to_keyword = 'To'
```

Comment:
While this *is* a bug, it should not be in this PR. I'll fix this myself. Thanks for pointing it out.